src/features/collaborator-avatars/CollaboratorListItem.scss

```diff
@@ -74,3 +74,7 @@
         }
     }
 }
+
+.bdl-CollaboratorListItem-user {
```

Comment:
I created a new class because the `user` class is on multiple levels in this component.

e.g. `collaborator-list-item > user > info > user`